test/e2e/test_cluster_configs/network_policy/antrea.json

```diff
@@ -0,0 +1,46 @@
+{
+	"env": {
+	},
+	"options": {
+		"allowedOrchestratorVersions": ["1.13", "1.14", "1.15", "1.16"]
```

Comment:
@reachjainrahul do @abhiraut do we know for sure that antrea doesn't work w/ k8s 1.17?

Reply:
in theory yes, it should work.. however let us update this only after we run some ci tests against 1.17? @jianjuns any opinion?